### ISSUE TITLE
Crash error – list index out of range 

### ISSUE THREAD

**Issue by laepage** [morisawausa/ProofingTool#23]
Hello,

I just tried to proof a font with your promising tool, and I received the following error. I'm using the latest Glyphs (3.3, 3309)

The Plugin ProofingTool.glyphsPlugin has caused a crash. Please update it by opening the Plugin Manager or re-downloading it from the original source.

Details:
Traceback (most recent call last):

  File "vanillaBase.py", line 539, in action_
    self.callback(sender)

  File "parameters.py", line 344, in triggerProofUpdate
    self.tryRerender()

  File "parameters.py", line 506, in tryRerender
    if self.parametersChanged(newParamSet, newGlyphSet):
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "parameters.py", line 485, in parametersChanged
    glyphsChanged = parameters['glyphs'][0] != self.parameters['glyphs'][0]
                    ~~~~~~~~~~~~~~~~~~~~^^^

IndexError: list index out of range

App:3.3-3309 Plugin:(null)-(null) System:10.15.7

**Comment by marieotsuka** [comment]
Hi @laepage, thanks for raising the issue and sorry for the delayed response to this. I hadn’t tested it with the cutting-edge version of Glyphs, but I’ve updated and now pushed a possible fix; it is working on Glyphs 3.3 (3311) on my end. 

I think it’s related to an issue described on the [forum here](https://forum.glyphsapp.com/t/plugin-menu-items-showing-as-nsmenuitem-in-the-app-ui-and-not-working/30289). 

You will need to remove and re-install the plugin and do a restart. If that doesn’t work, you may need to re-install the python module (which will unfortunately force you to remove and re-install all plugins.)

Let me know if the issue persists!

**Comment by laepage** [comment]
Hello @marieotsuka, no problem at all and thank you very much for the update!

I reinstalled the plugin (and the Vanilla and the DrawBot plugin to be on the safe side). I got the same crash twice, once the plugin crashed hard with a Glyphs exit, but now it's working. 

I was under the impression that the bug only occurs when the _Refresh glyphs_ option is checked, but now it seems to work with that on or off. I didn't reinstall the entire Python module, thinking I might be able to get by without it...
